insight_api/postgres/Review.py

```diff
@@ -0,0 +1,18 @@
+from insight_api.postgres.AsyncPostgresConnection import AsyncPostgresConnection
+
+
+async def find_top_rated_books(minimum_ratings, presented_amount=None):
+    connection = await AsyncPostgresConnection().get_connection()
+
+    optional_query_limitation, optional_query_argument = ("LIMIT $2", [presented_amount]) if presented_amount else (
```

Comment:
seperate the embedded if for presented amount to  a different line so it is less complex for the eyes